.github/workflows/pr.yml

```diff
@@ -86,6 +86,8 @@ jobs:
           fi
       - name: Run Go Tests
         run: make test
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v2
```

Comment:
Why is this not `v2.1.0` like in `codecov.yml`?

Reply:
Because I didn't want to pin the version in any of them but I forgot to remove it in one of them. Fixing it.